libs/dal/src/repositories/notification-template/notification-template.schema.ts

```diff
@@ -66,10 +66,11 @@ const notificationTemplateSchema = new Schema(
             children: [
               {
                 field: Schema.Types.String,
-                value: Schema.Types.String,
+                value: Schema.Types.Mixed,
                 operator: Schema.Types.String,
                 on: Schema.Types.String,
                 webhookUrl: Schema.Types.String,
+                timeOperator: Schema.Types.String,
               },
             ],
```

Comment:
I'm not sure what to do with the `children` schema, because we have right now 4 different filter types. Some of these types would never have some fields set, for ex. `isOnline` filter will be like:
```
{
   on: 'isOnline',
   value: true,
}
```